_topic_map.yml

```diff
@@ -813,6 +813,16 @@ Topics:
   - Name: Configuring ingress cluster traffic using a NodePort
     File: configuring-ingress-cluster-traffic-nodeport
     Distros: openshift-enterprise,openshift-webscale,openshift-origin
+# Kubernetes NMState (TECHNOLOGY PREVIEW)
+- Name: Kubernetes NMState (Tech Preview)
```

Comment:
I don't think we call out Tech Preview in navigation. We use the technology-preview.adoc to consistently state what is in Technology Preview.